lib/asciidoctor-diagram/ditaa/extension.rb

```diff
@@ -20,18 +21,16 @@ module Ditaa
           'transparent' => lambda { |o, v| o  << '--transparent' if v == 'true'}
       }
 
-      JARS = ['ditaamini-0.10.jar'].map do |jar|
-        File.expand_path File.join('../..', jar), File.dirname(__FILE__)
-      end
-      Java.classpath.concat JARS
-
       def self.included(mod)
         mod.register_format(:png, :image) do |parent, source|
           ditaa(parent, source)
         end
       end
 
       def ditaa(parent, source)
+
+        ditaa_jar = which_jar(parent, source, "ditaa", :raise_on_error => false)
```

Comment:
Shouldn't we raise an error if ditaa can not be found? Otherwise you'll get pretty obscure errors (NoClassDefFoundError probably) from the Java wrapper code.